Pods/Highlightr/Pod/Classes/HTMLUtils.swift

```diff
@@ -0,0 +1,294 @@
+
+import Foundation
+
+//Adapted from: https://gist.github.com/jrmgx/3f9f1d330b295cf6b1c6
+
+private let characterEntities : [String: Character] = [
```

Comment:
Does this duplicate anything from the HTMLString pod?

Reply:
I think it does but I haven't verified it.